source/getting_started/build-source.rst

```diff
@@ -149,53 +142,90 @@ Build the OS image
+   |C| source files and serve as the working directory, and to bring down the
+   latest version of :command:`repo` tool, including its most recent fixes.
+
+   .. note::
+      The URL specifies the manifest that refers to various repositories
+      used by |C|, which are placed within the working directory. For now, a
+      :file:`.repo/` folder is created to store the manifest and the metadata of
+      the source repositories.
 
    .. code-block:: bash
 
-       $ lunch celadon-userdebug
+       $ mkdir cic
+       $ cd cic
+       $ repo init -u https://github.com/projectceladon/manifest -b celadon/p/mr0/master -m cic
 
-#. The following command selects :command:`celadon_ivi-userdebug` as the
-   lunch target for building the
-   `Android 10 <https://www.android.com/android-10/>`_ Pre-Production
-   Release image with IVI UI:
+#. Enter the following command to pull down the |C| Android source tree to
+   your working directory. The :command:`repo sync` operation might take time
+   depending on your Internet download speed. Refer to the
+   `Downloading the Source <https://source.android.com/setup/build/downloading>`_
+   section of the AOSP website for tips to download the source behind a
+   proxy.
 
    .. code-block:: bash
 
-       $ lunch celadon_ivi-userdebug
+       $ repo sync -c
+
+.. _build-cic-package:
 
-   Alternatively, you can :ref:`run-as-service`; use either
-   :command:`caas-userdebug` or :command:`cic-userdebug` lunch targets as
-   shown below:
+Build the CIC package
+=====================
+
+#. Optionally, delete existing output of any previous build with the
+   following command in the top-most |C| source directory:
 
    .. code-block:: bash
 
-       $ lunch caas-userdebug
+       $ make clobber
 
-   or
+#. Enter the following command to initialize the build variables with the
+   :file:`envsetup.sh` script:
 
    .. code-block:: bash
 
-       $ lunch cic-userdebug
+       $ source build/envsetup.sh
 
-#. Build the |C| installer files with the command shown below.
+#. Specify your |C| lunch target using the :command:`lunch` command. You can
+   run :command:`lunch` with no arguments to choose different build
+   variants, and select a lunch target from a list of available options.
+   For example, the following commands configure the build system to
+   build the package containing |C| in Container images with the traditional tablet UI:
 
    .. code-block:: bash
 
-       $ make SPARSE_IMG=true flashfiles -j $(nproc)
-    
-   .. note::
-      The *-j $(nproc)* argument instructs the builder to compile the source
-      code with parallel tasks. The generated kernelflinger executables
-      .ZIP file
-      (:file:`out/target/product/celadon/celadon.flashfiles.eng.${USER}.zip`)
-      is available after the build. You can follow :ref:`install-on-nuc` of
-      this guide to flash the installer image to a removable USB drive and
-      install |C| on a |NUC|.
+       $ lunch cic-dev
```

Comment:
cic-dev -> cic_dev-userdebug

Reply:
@swaroopbalan @JeevakaPrabu Thanks for feedbacks, they are incorporated. If you are all OK with the current commit, it should be published on 01.org after 8AM PST.

Reply:
Yes , i am OK